app/scripts/modules/core/delivery/create/CreatePipelineModal.tsx

```diff
@@ -112,40 +115,40 @@ export class CreatePipelineModal extends React.Component<ICreatePipelineModalPro
         this.props.pipelineSavedCallback(newPipeline.id);
       }
     });
-  };
+  }
 
-  private onSaveFailure = (response: IHttpPromiseCallbackArg<{message: string}>): void => {
+  private onSaveFailure(response: IHttpPromiseCallbackArg<{message: string}>): void {
     $log.warn(response);
     this.setState({
       submitting: false,
       saveError: true,
       errorMessage: (response && response.data && response.data.message) || 'No message provided',
     });
-  };
+  }
 
-  public close = (): void => {
+  public close(): void {
     this.setState(this.getDefaultState());
     this.props.showCallback(false);
-  };
+  }
 
-  private handleTypeChange = (option: Select.Option): void => {
+  private handleTypeChange(option: Select.Option): void {
     this.setState({command: Object.assign({}, this.state.command, {strategy: option.value})});
-  };
+  }
 
-  private handleNameChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+  private handleNameChange(e: React.ChangeEvent<HTMLInputElement>): void {
     this.setState({command: Object.assign({}, this.state.command, {name: e.target.value})});
-  };
+  }
 
-  private handleTemplateChange = (option: Select.Option): void => {
+  private handleTemplateChange(option: Select.Option): void {
     const template = this.state.templates.find(t => t.name === option.value);
     this.setState({command: Object.assign({}, this.state.command, {template})});
-  };
+  }
 
-  private handleErrorDismiss = (): void => {
+  private handleErrorDismiss(): void {
     this.setState({saveError: false});
-  };
+  }
 
-  private templateOptionRenderer = (option: Select.Option) => {
+  private templateOptionRenderer(option: Select.Option) {
```

Comment:
I'm on board with omitting return types of void, but this returns `React.Element[]` or something, right?

(I know you were only updating the autobind bits, just noticed this now and while you're in there...)